src/test/deploy/functions/runtimes/node/parseTriggers.spec.ts

```diff
@@ -74,8 +78,8 @@ describe("addResourcesToBuild", () => {
       ...BASIC_BACKEND_ENDPOINT,
       httpsTrigger: {},
     });
-    const convertedBackend: backend.Backend = build.resolveBackend(expected, {});
-    expect(convertedBackend).to.deep.equal(expectedBackend);
+    const convertedBackend = resolveBackend(expected);
+    expect(convertedBackend).to.eventually.deep.equal(expectedBackend);
```

Comment:
When an expectation is called with an `eventually` modifier it returns a promise you need to await.

Reply:
Fixed, I think.